src/util.ts

```diff
@@ -48,3 +48,27 @@ export async function runNpm(
     args: string[], options?: ExecFileOptions): Promise<string|Buffer> {
   return promisify(execFile)(npmCmd, args, options).then(({stdout}) => stdout);
 }
+
+/**
+ * Escape a string such that it can be safely embedded in a JavaScript template
+ * literal (backtick string).
+ */
+export function escapeStringLiteral(unescaped: string): string {
```

Comment:
If this is only used in one place, consider putting it near there for now, rather than in util, since it's more specific than for a general string literal